docs/contributing.rst

```diff
@@ -1,50 +1,170 @@
 Contributing to tedana
 ======================
 
-This document explains how to set up a development environment for contributing
-to tedana and code style conventions we follow within the project.
-For a more general guide to the tedana development, please see our
-`contributing guide`_. Please also follow our `code of conduct`_.
+This document explains contributing to ``tedana`` at a very high level,
+with a focus on project governance and development philosophy.
+For a more practical guide to the tedana development, please see our
+`contributing guide`_.
 
 .. _contributing guide: https://github.com/ME-ICA/tedana/blob/master/CONTRIBUTING.md
-.. _code of conduct: https://github.com/ME-ICA/tedana/blob/master/CODE_OF_CONDUCT.md
 
+Governance
+----------
 
-Style Guide
-===========
+Governance is a hugely important part of any project.
+It is especially important to have clear process and communication channels
+for open source projects that rely on a distributed network of volunteers, such as ``tedana``.
 
-Code
-````
+```tedana`` is currently supported by a small group of five core developers.
+Even with only five members involved in decision making processes,
+we've found that setting expectations and communicating a shared vision has great value.
 
-Docstrings should follow `numpydoc`_ convention. We encourage extensive
-documentation.
+By starting the governance structure early in our development,
+we hope to welcome more people into the contributing team.
+We are committed to continuing to update the governance structures as necessary.
+Every member of the ``tedana`` community is encouraged to comment on these processes and suggest improvements.
 
-The code itself should follow `PEP8`_ convention as much as possible, with at
-most about 500 lines of code (not including docstrings) per script.
+Code of conduct
+```````````````
 
-.. _numpydoc: https://numpydoc.readthedocs.io/en/latest/format.html
-.. _PEP8: https://www.python.org/dev/peps/pep-0008/
+All ``tedana`` community members are expected to follow our `code of conduct`_
+during any interaction with the project.
+That includes---but is not limited to---online conversations,
+in-person workshops or development sprints, and when giving talks about the software.
 
-Pull Requests
-`````````````
+As is stated in the code, severe or repeated violations by community members may result in exclusion
```

Comment:
```suggestion
As it is stated in the code, severe or repeated violations by community members may result in exclusion
```

Reply:
I'm not sure I follow the suggested change here - but I think the fact that @yarikoptic doesn't read the sentence the same way I do is clear that it needs a rework!

I think closer to what I was intending is:

```suggestion
As stated in the code, severe or repeated violations by community members may result in exclusion
```